pkg/virt-controller/watch/vm.go

```diff
@@ -924,6 +935,79 @@ func (c *VMController) stopVMI(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMac
 	return nil
 }
 
+func vmRevisionNamePrefix(vmName string) string {
+	return fmt.Sprintf("revision-start-vm-%s", vmName)
```

Comment:
is it safe to use name instead of uid?  Could there somehow be a name conflict on delete/recreate?

Reply:
it shouldnt have been a problem as far as I thought but I changed it anyways to uid